.gitignore

```diff
@@ -0,0 +1,5 @@
+._*
+.~lock.*
+.buildpath
```

Comment:
Consider removing these in a subsequent PR: no need for environment-specific ignores